sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/InMemoryTableScanExec.scala

```diff
@@ -222,6 +225,8 @@ private[sql] case class InMemoryTableScanExec(
     @transient relation: InMemoryRelation)
   extends LeafExecNode {
 
+  override protected def innerChildren: Seq[TreeNode[_]] = Seq(relation) ++ super.innerChildren
```

Comment:
ok seems this is the only special case, how about we add a comment to explain why we call `super.innerChildren` here? e.g. There are expressions inside this operator and may contain subquery expression.